Mathlib/Analysis/Normed/Ring/Seminorm.lean

```diff
@@ -388,3 +388,52 @@ lemma MulRingNorm_nat_le_nat {R : Type*} [Ring R] (n : ℕ) (f : MulRingNorm R)
       f (n + 1) ≤ f (n) + f 1 := f.add_le' ↑n 1
       _ = f (n) + 1 := by rw [map_one]
       _ ≤ n + 1 := add_le_add_right hn 1
+
+noncomputable section
+variable {R : Type*} [Ring R]
+variable {f g : MulRingNorm ℚ}
+
+/-- A multiplicative norm composed with the absolute value on integers equals the norm itself. -/
+lemma f_of_abs_eq_f (x : ℤ) (f : MulRingNorm ℚ) : f (Int.natAbs x) = f x := by
```

Comment:
This should be renamed. Maybe to `MulRingNorm.apply_natAbs_eq`?

Reply:
Also, this should work for `f : MulRingNorm R` over any ring `R`.

Reply:
I like your suggestion. As for the generalization to any R, then that'd justify the variable R :D